plutus-playground-lib/test/Playground/APISpec.hs

```diff
@@ -4,49 +4,54 @@ module Playground.APISpec
     ( spec
     ) where
 
-import           Language.Haskell.Interpreter (CompilationError (CompilationError, RawError), column, filename, row,
-                                               text)
-import           Playground.API               (parseErrorText)
-import           Test.Hspec                   (Spec, describe, it, shouldBe)
+import           Data.Proxy      (Proxy (Proxy))
+import           Data.Swagger    (toInlinedSchema)
+import           Ledger.Interval (Interval)
+import           Ledger.Value.TH (Value)
+import           Playground.API  (SimpleArgumentSchema (SimpleArraySchema, SimpleIntSchema, SimpleObjectSchema, SimpleStringSchema, SimpleTupleSchema, ValueSchema),
+                                  isSupportedByFrontend, toSimpleArgumentSchema)
+import           Test.Hspec      (Spec, describe, it, shouldBe)
 
 spec :: Spec
-spec = parseErrorTextSpec
+spec = do
+    toSimpleArgumentSchemaSpec
+    isSupportedByFrontendSpec
 
-parseErrorTextSpec :: Spec
```

Comment:
where have these tests gone?

Reply:
Ah, good question. They were duplicated, so I deleted the duplicate. But looking again I realise that their best home would be merged into this file. Corrected. :-)